src/Card.tsx

```diff
@@ -284,95 +363,210 @@ export default function Card({
+                  </button>
+                  <button 
+                    className="p-1 px-2 rounded-full transition-all"
+                    style={draftCardType == "repeater" ? {
+                      background: THEMES[theme].background,
+                      color: THEMES[theme].color
+                    } : {
+                      background: "none",
+                      color: THEMES[theme].background
+                    }}
+                    onClick={() => setDraftCardType("repeater")}
+                  >
+                    <FontAwesomeIcon icon={faTowerBroadcast}/>
+                  </button>
+                </div>
+              ) : (
+                contact.cardType == "person" ? (
+                  <FontAwesomeIcon icon={faPerson} className="text-4xl"/>
+                ) : (
+                  <FontAwesomeIcon icon={faTowerBroadcast} className="text-3xl"/>
+                )
+              )}        
+            </div>
+          </div>
+          {(editMode && draftCardType == "person") || (!editMode && contact.cardType == "person") ? (
+            <Input
+              className="text-3xl -my-1"
+              value={displayName}
+              onChange={(e) => setDraftName(e.target.value)}
+              onKeyDown={handleInputKeyDown}
+              placeholder="name"
+              disabled={!editMode}
+            />
+          ) : null} 
+          {(editMode && draftCardType == "repeater") || (!editMode && contact.cardType == "repeater") ? (
+            <Input
+              className="text-3xl -my1"
+              value={displayLocation}
+              onChange={(e) => setDraftLocation(e.target.value)}
+              onKeyDown={handleInputKeyDown}
+              placeholder="location"
+              disabled={!editMode}
+            />     
           ) : null}
-          {(contact.website || editMode) && (
-            <Field label="WEB">
-              {createMode || editMode ? (
-                <div className="w-36">
+
+        </div>
+        {(editMode && draftCardType == "repeater") || (!editMode && contact.cardType == "repeater") ? (
+          <div className="z-10 text-sm w-52">
+            <div className="grid grid-cols-2 gap-0">
+              {editMode || contact.frequency ? (
+                <Input
+                  className=""
+                  value={displayFreq}
+                  onChange={(e) => {setDraftFrequency(e.target.value)}}
+                  onKeyDown={handleInputKeyDown}
+                  placeholder="frequency"
+                  disabled={!editMode}
+                />
+              ) : null}
+              {editMode || contact.offset ? (
+                <Input
+                  className=""
+                  value={displayOffset}
+                  onChange={(e) => {setDraftOffset(e.target.value)}}
+                  onKeyDown={handleInputKeyDown}
+                  placeholder="offset"
+                  disabled={!editMode}
+                />
```

Comment:
what did you end up deciding on representing cross-band repeaters/satellites? seems like this only handles same/band

Reply:
I didn't, my thought is that offset could be offset/tx freq

Reply:
maybe another field? but then it feels like its really bloated...